### PR TITLE
feat: Add Factorial method to Maths.cs

### PR DESCRIPTION
**Pull request by RogerMo01**

### CODE REVIEW

File: Recursive Exercices/Recursive Exercices/Maths.cs [hunk 1]
```diff
@@ -8,5 +8,9 @@ namespace Recursive_Exercices
 {
     class Maths
     {
+        public static int Factorial(int number)
+        {
+            return (number == 1) ? 1 : number * Factorial(number - 1);
```
Comment (LongJohnSilver1504) [inline Recursive Exercices/Recursive Exercices/Maths.cs:13]: What about 0?

Comment (RogerMo01) [reply Recursive Exercices/Recursive Exercices/Maths.cs]: Fixed

**Review by LongJohnSilver1504 — APPROVED** [review]
LGTM!

**Comment by LongJohnSilver1504** [comment]
PLease resolve the conflict

**Pull request merged**